hwproj.front/src/components/Experts/InviteModal.tsx

```diff
@@ -309,20 +323,25 @@ const InviteExpertModal: FC<IInviteExpertProps> = (props) => {
                                 Закрыть
                             </Button>
                         </Grid>
-                        <Grid item>
+                        {!isInvited && <Grid item>
                             <Button
                                 variant="contained"
                                 color="primary"
                                 onClick={handleInvitation}
                                 disabled={isInviteButtonDisabled}
                             >
-                                Пригласить
+                                Получить ссылку для приглашения
```

Comment:
```suggestion
                                Получить ссылку
```
Остальное из контекста понятно